contexts/notification-context.provider.tsx

```
@@ -0,0 +1,98 @@
+      const unreadNotifications = notifications.filter((notification) => !notification.isRead);
+
+      setUnreadNotifications(unreadNotifications.length);
+
+      setUserNotifications(notifications);
+    });
+  }
+
+  useEffect(() => {
+    let unsubscribe: (() => void) | undefined;
+
+    if (user && user.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)